src/openrct2/network/Socket.cpp

```diff
@@ -504,6 +504,11 @@ class TcpSocket final : public ITcpSocket, protected Socket
             throw std::runtime_error("Socket not closed.");
         }
 
+        // The status is  When connect is called, status is set to resolving but we want to make sure
+        // the status is changed before this async method exits otherwise the consumer
+        // might think the status has closed before it started to connect
```

Comment:
```suggestion
        // When connect is called, the status is set to resolving, but we want to make sure
        // the status is changed before this async method exits, otherwise the consumer
        // might think the status has closed before it started to connect.
```

Reply:
Agreed with @Broxzier, although I think "otherwise" should start its own sentence:
```suggestion
        // When connect is called, the status is set to resolving, but we want to make sure
        // the status is changed before this async method exits. Otherwise, the consumer
        // might think the status has closed before it started to connect.
```